raiden/transfer/state.py

```diff
@@ -456,11 +459,12 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'address': to_checksum_address(self.address),
             'token_address': to_checksum_address(self.token_address),
             'network_graph': self.network_graph,
-            'partneraddresses_to_channels': map_dict(
+            'channelidentifiers_to_channels': map_dict(
                 to_checksum_address,
                 serialization.identity,
-                self.partneraddresses_to_channels,
+                self.channelidentifiers_to_channels,
             ),
+            'partneraddresses_to_channelidentifiers': self.partneraddresses_to_channelidentifiers,
```

Comment:
But here a `map_dict` is needed as the `partneraddress` is bytes and needs to be serialized to a string:
```py
map_dict(to_checksum_address, serialization.identity, self.partneraddresses_to_channelidentifiers)
```

This also requires changes in the deserialization